src/libexpr/primops/fetchTree.cc

```diff
@@ -56,6 +56,13 @@ void emitTreeAttrs(
 
     }
 
+    // `dirtyRev` does not exist only when there are no commits
+    // OR when `builtins.getFlake` is called
```

Comment:
I'm not sure what that comment means: `dirtyRev` should only exist when fetching a dirty local repo, right?

Reply:
Yeah, I originally added this comment just to help me when writing this PR, I'm happy to remove it if you think it'll be less confusing

Reply:
Yeah, I think it would be better to drop it. Just ping me once it's done and I can merge

Reply:
@thufschmitt updated :+1:

Reply:
Thanks :)